assets/model_monitoring/components/src/model_monitor_evaluate_metrics_threshold/evaluate_metrics_threshold.py

```diff
@@ -51,6 +50,10 @@ def evaluate_metrics_threshold(
         post_warning_event(error_message)
         if notification_emails is not None and notification_emails != "":
             post_email_event(signal_name, notification_emails, error_message)
+        add_tags_to_run(_get_run_id(),
+                        {
+                            f"azureml_modelmonitor_threshold_breached": error_message
```

Comment:
Did you check with PM if this key is good or not? Currently, the key doesn't have any signal information.